Code/Tools/ProjectManager/Source/DownloadController.h

```diff
@@ -57,16 +70,16 @@ namespace O3DE::ProjectManager
         void HandleResults(const QString& result, const QString& detailedError);
 
     signals:
-        void StartGemDownload(const QString& gemName, bool downloadNow);
+        void StartObjectDownload(const QString& gemName, DownloadObjectType objectType, bool downloadNow);
         void Done(const QString& gemName, bool success = true);
-        void GemDownloadAdded(const QString& gemName);
-        void GemDownloadRemoved(const QString& gemName);
-        void GemDownloadProgress(const QString& gemName, int bytesDownloaded, int totalBytes);
+        void ObjectDownloadAdded(const QString& ObjectName, DownloadObjectType objectType);
+        void ObjectDownloadRemoved(const QString& ObjectName, DownloadObjectType objectType);
+        void ObjectDownloadProgress(const QString& objectName, DownloadObjectType objectType, int bytesDownloaded, int totalBytes);
```

Comment:
Some of these params are using different casing
```suggestion
        void ObjectDownloadAdded(const QString& objectName, DownloadObjectType objectType);
        void ObjectDownloadRemoved(const QString& objectName, DownloadObjectType objectType);
        void ObjectDownloadProgress(const QString& objectName, DownloadObjectType objectType, int bytesDownloaded, int totalBytes);
```